src/main/java/com/ing/data/cassandra/jdbc/utils/ArrayImpl.java

```diff
@@ -0,0 +1,76 @@
+package com.ing.data.cassandra.jdbc.utils;
+
+import java.sql.Array;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.SQLFeatureNotSupportedException;
+import java.util.Map;
+
+/**
+ * @author Liudmila Kornilova
```

Comment:
The authorship is not required here as git tracking is sufficient, but you can add a quick description of the purpose of this class in the Javadoc.

Also, the [license header](https://github.com/ing-bank/cassandra-jdbc-wrapper/blob/release/next/CONTRIBUTING.md#license-headers) must be added to any new file to conform to the contributing guidelines.